src/tpl/create-account.html

```diff
@@ -27,8 +27,265 @@ <h2 class="typo-title">Create Whiteout account</h2>
             <input class="input-text right" ng-model="confirmPass" required type="password" placeholder="Confirm password" ng-class="{'input-text--error': (pass || confirmPass) && pass !== confirmPass}" tabindex="4">
           </div>
         </div>
-        <div class="form__row">
-          <input class="input-text"  type="tel" ng-model="phone" required placeholder="Mobile phone number (e.g. +49 170 1234567)" tabindex="5">
+        <div class="form__row form__row--multi">
+          <div class="form__col" style="max-width:130px">
```

Comment:
I had to add this inline style here since flex-box did not work for some reason. @webholics can you take a look what I'm doing wrong? thanks